examples/hello-world/src/cli/index.ts

```diff
@@ -0,0 +1,26 @@
+import colors from 'colors';
+import { Command } from 'commander';
+import { Model } from '../language-server/generated/ast';
+import { HelloWorldLanguageMetaData } from '../language-server/generated/meta-data';
+import { createHelloWorldServices } from '../language-server/hello-world-module';
+import { extractAstNode } from './cli-util';
+import { HelloWorldGenerator } from './generator';
+const metaData = new HelloWorldLanguageMetaData();
```

Comment:
Instead of instantiating a new meta data object here, we should use the one provided by the `LangiumServices`.